character_dnd.py

```diff
@@ -0,0 +1,171 @@
+import math
+from random import randint
+
+
+class Character_DnD:
```

Comment:
@jimbobway40 You can comment inline with questions

Reply:
Got it.